CONTRIBUTING.md

```diff
@@ -17,6 +17,11 @@ Scanamo uses a standard [SBT](https://www.scala-sbt.org/) build. If you
 have SBT installed you can just run the `test` command from the SBT prompt
 to compile Scanamo and run its tests.
 
+Some tests require that a local dynamodb instance is running (at port 8042). You can download dynamodb from [this website](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html). How to run local dynamodb at a certain port is shown below:
```

Comment:
Thanks a lot @kiranbayram 🙌   in fact there is a `startDynamoDbLocal` task that will do this for you. If you amend this change I'm happy to roll this out 🎉

Reply:
Ah sorry, didn't see that. I'll change the commit shortly